scripts/tools.sh

```diff
@@ -10,5 +10,5 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 cd "$SCRIPT_DIR"
 
 ./install-gotestsum.sh -b $(go env GOPATH)/bin
+./install-goreleaser.sh -b $(go env GOPATH)/bin v0.182.1
```

Comment:
Chybu v instalacii `GoReleaser` sice fixli rychlo ale blbo:
https://github.com/goreleaser/goinstall/commit/b7efd25ecda86fca22ee6358c4e2e9263e35c08b

Nie je to spatne kompatibilne, tak do repa pridavam instalacny script, nech mame nad tym kontrolu.